.github/ISSUE_TEMPLATE/question.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: ❓ Question
+description: Share a question about WandB
+title: "[Q] "
+
+body:
+
+  - type: markdown
+    attributes:
+      value: >-
+        We are always happy to hear from our users.
```

Comment:
@cvphelps should we link here to other places, where we take questions. Also we can add a `config.yml` with additional links.